From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Fri, 19 Mar 2021 09:50:46 +0100
Subject: [PATCH] multipathd: reduce log levels in cli_add_map()

Normally "add map" will be used to add a map which doesn't exist
yet. Thus not finding this map in the first place is not a problem
indicator and should be logged at level 3 only.

Signed-off-by: Benjamin Marzinski <bmarzins@redhat.com>
---
 multipathd/cli_handlers.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/multipathd/cli_handlers.c b/multipathd/cli_handlers.c
index 7f3e61f6..1de6ad8e 100644
--- a/multipathd/cli_handlers.c
+++ b/multipathd/cli_handlers.c
@@ -852,14 +852,15 @@ cli_add_map (void * v, char ** reply, int * len, void * data)
 	}
 	do {
 		if (dm_get_major_minor(param, &major, &minor) < 0)
-			condlog(2, "%s: not a device mapper table", param);
+			condlog(count ? 2 : 3,
+				"%s: not a device mapper table", param);
 		else {
 			sprintf(dev_path, "dm-%d", minor);
 			alias = dm_mapname(major, minor);
 		}
 		/*if there is no mapname found, we first create the device*/
 		if (!alias && !count) {
-			condlog(2, "%s: mapname not found for %d:%d",
+			condlog(3, "%s: mapname not found for %d:%d",
 				param, major, minor);
 			get_refwwid(CMD_NONE, param, DEV_DEVMAP,
 				    vecs->pathvec, &refwwid);
